ibis/backends/clickhouse/tests/conftest.py

```diff
@@ -27,6 +28,72 @@ class TestConf(UnorderedComparator, BackendTest, RoundHalfToEven):
     supports_floating_modulus = False
     bool_is_int = True
 
+    @staticmethod
+    def load_data(data_dir: Path, script_dir: Path, **kwargs) -> None:
+        """Load testdata into a clikhouse backend.
+
+        Parameters
+        ----------
+        data_dir : Path
+            Location of testdata
+        script_dir : Path
+            Location of scripts defining schemas
+        """
+        import clickhouse_driver
+
+        host = (
```

Comment:
`kwargs.get("host", CLICKHOUSE_HOST)` should do the same, or you could explicitly define these as default keyword arguments.